docs/source/api.md

```diff
@@ -0,0 +1,17 @@
+# API reference
```

Comment:
What do you think about using RST files as the standard under `docs/source` similar to GeoPandas?  I don't have a strong preference, but it seems like this would sidestep the need to add the `{eval-rst}` context here.

Reply:
I have changed to have this one a pure RST. Since there's no text it doesn't matter so much.